internal/provider/provider.go

```diff
@@ -127,9 +128,12 @@ func (p *BloxOneProvider) Resources(_ context.Context) []func() resource.Resourc
 
 		keys.NewTsigResource,
 
+		anycast.NewOnPremAnycastManagerResource,
```

Comment:
The name of the resource here is inconsistent.

Reply:
Please suggest new name

Reply:
done